src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -46,11 +50,17 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
 }: ReviewChangeRequestViewProps) => {
   const [selected, setSelected] = useState(-1);
   const toast = useToast();
-
+  const { isLoading, isError, error, data } = useGetBlockingWorkPackages(cr.wbsNum);
+  const [showWarning, setShowWarning] = useState(false);
+  const [formData, setFormData] = useState<FormInput>({ reviewNotes: '', accepted: false, psId: '' });
```

Comment:
why do we now have two use states for this? (useForm does exactly this, this defeats the purpose of using react-hook-forms)

Reply:
I dont remember writing this but uhhh yeah thats weird